app/javascript/flavours/glitch/initial_state.js

```diff
@@ -81,14 +82,19 @@
  * @property {boolean=} use_pending_items
  * @property {string} version
  * @property {boolean} translation_enabled
- * @property {object} local_settings
+ * @property {string} status_page_url
```

Comment:
`status_page_url` should be an upstream thing… I'll investigate.

Reply:
It is.
Upstream didn't add it to the typedef.
I can replace it with an expect error comment, if you want to.